orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepository.kt

```diff
@@ -667,7 +667,10 @@ class SqlExecutionRepository(
         .join(
           jooq.selectExecutions(
           PIPELINE,
-          listOf(field("id")),
+          fields = if (compressionProperties.enabled)
```

Comment:
What if compression is later disabled?

Reply:
I'm not sure there's a good way to identify this case or handle it, just thinking...